packages/web-components/fast-foundation/src/listbox/listbox.element.ts

```diff
@@ -0,0 +1,114 @@
+import {
+    attr,
+    css,
+    ElementStyles,
+    nullableNumberConverter,
+    observable,
+} from "@microsoft/fast-element";
+import { Listbox } from "./listbox";
+
+/**
+ * A Listbox Custom HTML Element.
+ * Implements the {@link https://w3c.github.io/aria/#listbox | ARIA listbox }.
+ *
+ * @public
+ */
+export class ListboxElement extends Listbox {
+    /**
+     * The attribute to set the size property.
+     *
+     * @remarks
+     * HTML Attribute: `size`.
+     *
+     * The value of the `size` attribute is reflected by the `size` property.
+     *
+     * @public
+     */
+    @attr({
+        attribute: "size",
+        converter: nullableNumberConverter,
+        mode: "fromView",
+    })
+    public sizeAttribute: number;
+
+    /**
+     * The number of options to display.
+     *
+     * @remarks
+     * Setting `size` property does not change the value via the `size` attribute.
+     *
+     * Any value above zero will force the component to display as a listbox.
+     *
+     * @public
+     */
+    @observable
+    public size: number = 0;
```

Comment:
I think I saw a question recently relating to a similar relationship of something which was an attribute and the property didn't map directly. If I want to programmatically set size, setting this will NOT affect the attribute - can you elaborate a bit on your thoughts here?

Reply:
I wasn't sure if this was convention but it appears that most browsers do bind the `size` attribute to the `size` property. I'll push an update to simplify this.

Reply:
Update: it was due to the interactions between `multiple` and `size`. When `multiple` is present, the `size` property defaults to `4`. This default is set internally, and isn't reflected in the `size` attribute with a native `<select>` element.

Reply:
Gotcha - do you think we'll hit issues where a user expects `myListbox.setAttribute("size", "4")` and `myListbox.size = 4` to both update the same thing? Is there a way we can address that or ensure the API works as expected?

Reply:
This PR doesn't deal with the `multiple` case, so I updated it to handle the `size` property and attribute as one `@attr`-decorated property. Changing the property now updates the attribute and vice versa.